A/AMCL/build_tarballs.jl

```diff
@@ -0,0 +1,107 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "AMCL"
+version = v"2.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/apache/incubator-milagro-crypto-c/archive/$version.tar.gz", "48fc7e0d7ad5edbfe8a779dc56f28813c25798ddfd60f1d94ccf1203eacd5645"),
+]
+
+# Bash recipe for building across all platforms
+script = """
+cd "\$WORKSPACE/srcdir/incubator-milagro-crypto-c-$version"
+""" + raw"""
+set -e
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="$prefix" -DBUILD_BENCHMARKS=off -DBUILD_DOCS=off -DBUILD_EXAMPLES=off -DBUILD_PYTHON=off ..
+make -j${nproc}
+make test
```

Comment:
I assume this will try to run some programs compiled with the cross-compiler?  unfortunately we can't do that, as we don't currently have an emulator.  For this reason, you'll probably need to remove this step.

Reply:
Understood, removing tests.